01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,12 @@
+// CommonJS, every file is module (by default)
+// Modules  - Encapsulated Code (only share minimum)
+
+const names = require('./04-names') // .js is optional
+const sayHi = require('./05-utils')
+const data = require('./06-althernative-flavor')
+require('./07-mind-grenade')    // per the omported module, addValues function is called
```

Comment:
Everything looks good just wanted to mention the o that was mistyped on imported.